scripts/genesis-validate.sh

```diff
@@ -44,18 +47,44 @@ fi
 
 
 # collect gentx
-$ARCHD collect-gentxs --log_level error --home $ARCHDIR
+echo "Collecting gentx"
+trap "$($ARCHD collect-gentxs --home $ARCHDIR >> $LOGS_FILE 2>&1)" EXIT
```

Comment:
I don't think trap is needed here, I would change it to
```
$ARCHD collect-gentxs --home $ARCHDIR >> $LOGS_FILE 2>&1 || true
```

Reply:
What is being trapped here? I assume the `collect-gentxs` will run, only the print output is being trapped till the end of the script?